### PR TITLE
Automatically generate sitemap.xml after building HTMLs

### PR DESCRIPTION
**Pull request by refeed**
Closes https://github.com/zulip/zulip-archive/issues/70

### CODE REVIEW

**Comment by rht** [comment]
Do you have a sample GH Pages instance to test this?

**Comment by refeed** [comment]
Not yet, gonna create it now

**Comment by refeed** [comment]
@rht just made one at https://refeed.github.io/zarchive-refeed/ the sitemap is in https://refeed.github.io/zarchive-refeed/sitemap.xml

**Comment by rht** [comment]
The sitemap.xml and the .gz looks good. The remaining thing is to add fullstop at the end of your commit messages.

**Comment by lukasstevens** [comment]
Is there any progress on this? This feature would be very useful.

**Comment by timabbott** [comment]
Merged, thanks @refeed! 

Perhaps you can add documentation of this feature in a follow-up PR?

**Comment by refeed** [comment]
Sure, will do this weekend!

**Comment by refeed** [comment]
Just opened a PR for the docs at https://github.com/zulip/zulip-archive/pull/92

File: entrypoint.sh [hunk 1]
```diff
@@ -32,8 +32,7 @@ python3 get-pip.py
 pip install virtualenv
 virtualenv -p python3 .
 source bin/activate
-pip3 install zulip==0.6.3
-pip3 install pyyaml==5.2
+pip3 install install -r requirements.txt
```
Comment (eric-wieser) [inline entrypoint.sh:35]: `install` is here twice: #96

**Review by 5twelve — APPROVED** [review]

**Review by 5twelve — APPROVED** [review]

**Pull request merged**